go/libraries/doltcore/mvdata/data_mover.go

```diff
@@ -170,6 +187,22 @@ func MoveDataToRoot(ctx context.Context, mover *DataMover, mvOpts DataMoverOptio
 		return nil, badCount, errhand.BuildDError("An error occurred moving data:\n").AddCause(err).Build()
 	}
 
+	if len(badRows) > 0 {
```

Comment:
If all you're doing with bad rows is printing them out, just do that in the callback. No need to return a slice of all of them here